src/guides/v2.3/graphql/queries/index.md

```diff
@@ -170,11 +174,72 @@ query IntrospectionQuery {
         "fields": [
           {
             "name": "cart",
-            "description": "Returns information about shopping cart"
+            "description": "Returns information about shopping cart",
+            "type": {
+              "name": "Cart",
+              "kind": "OBJECT"
+            }
           },
           {
             "name": "categoryList",
-            "description": "Returns an array of categories based on the specified filters."
+            "description": "Returns an array of categories based on the specified filters.",
+            "type": {
+              "name": null,
+              "kind": "LIST"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+#### Return a list of Magento mutations
```

Comment:
Since the information in this article is related to queries, I would suggest moving this section to the article that describes mutations 

https://devdocs.magento.com/guides/v2.3/graphql/mutations/index.html

Thank you.